### PR TITLE
Update create-app-side-panes.md

### PR DESCRIPTION
**Pull request by TheSarcasticGuy**
Added documentation for opening a web resource in sidepane.
app-side-panes-opening-webresource

### CODE REVIEW

**Comment by TheSarcasticGuy** [comment]
@microsoft-github-policy-service agree

File: powerapps-docs/developer/model-driven-apps/clientapi/create-app-side-panes.md [hunk 1]
```diff
@@ -91,6 +91,28 @@ Xrm.App.sidePanes.createPane({
...
+                    }).then((pane) => {
+                        //navigate to webresource
+                        pane.navigate({
+                            pageType: "webresource",
+                            webresourceName: "new_RegistrationPage",
+                        })
+                    });
+```
+> [!div class="mx-imgBorder"] 
+> ![Screenshot showing a webresource opened]<img width="291" alt="app-side-panes-opening-webresource" src="https://github.com/TheSarcasticGuy/powerapps-docs/assets/37799232/fbdc3fed-18f3-4be6-999f-47d1a032339c">
```
Comment (JimDaly) [inline powerapps-docs/developer/model-driven-apps/clientapi/create-app-side-panes.md:113]: @TheSarcasticGuy 

Thanks for your contribution.

I'm afraid we can't have links to external sites for images.
Can you place the image file in this folder: `powerapps-docs/developer/model-driven-apps/clientapi/media` and then include a relative link to it?

**Review by JimDaly — CHANGES_REQUESTED** [review]

**Comment by TheSarcasticGuy** [comment]
Hi Jim,Thank you for your suggestion. However, when I went to the said directory and tried to upload a file it threw a privilege issue stating that File Uploads require push access to this directory.  Below is the screenshot of the same. Could you please help me out here?Thanks in advanced!AjinkyaSent from my iPhoneOn 28 May 2023, at 10:16 PM, Jim Daly ***@***.***> wrote:﻿
@JimDaly requested changes on this pull request.



In powerapps-docs/developer/model-driven-apps/clientapi/create-app-side-panes.md:
> +```javascript
+Xrm.App.sidePanes.createPane({
+                        title: "Registration Page",
+                        imageSrc: "WebResources/msfp_SurveyIcon_32",
+                        paneId: "RegistrationPage",
+                        canClose: false
+                    }).then((pane) => {
+                        //navigate to webresource
+                        pane.navigate({
+                            pageType: "webresource",
+                            webresourceName: "new_RegistrationPage",
+                        })
+                    });
+```
+> [!div class="mx-imgBorder"] 
+> ![Screenshot showing a webresource opened]app-side-panes-opening-webresource

@TheSarcasticGuy
Thanks for your contribution.
I'm afraid we can't have links to external sites for images.
Can you place the image file in this folder: powerapps-docs/developer/model-driven-apps/clientapi/media and then include a relative link to it?

**Review by JimDaly — APPROVED** [review]

**Comment by JimDaly** [comment]
@TheSarcasticGuy 
I'll accept this PR and add the image from my side.
Thanks

**Pull request merged**